helm/vmss-prototype/templates/vmss-prototype.yaml

```diff
@@ -63,6 +92,7 @@ spec:
             {{- end }}
             {{- end }}
         spec:
+          serviceAccountName: kamino-{{ $jobName }}
```

Comment:
@Michael-Sinz is this sufficient to enable the service account for the Job/CronJob ?

Reply:
Yes

Reply:
However, you would remove the KUBECONFIG environment variable setting and the kubeconfig mount/volume bits.

Reply:
kubeconfig references removed